src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -212,7 +222,22 @@ public async Task<IActionResult> BulkEdit(WorkflowTypeIndexOptions options, IEnu
             return RedirectToAction(nameof(Index));
         }
 
-        public async Task<IActionResult> EditProperties(long? id, string returnUrl = null)
+        public async Task<IActionResult> Export(int id)
+        {
+            if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageWorkflows))
+            {
+                return Forbid();
+            }
+
+            var resultFileName = await ExportWorkflows(id);
+
+            return new PhysicalFileResult(resultFileName, "application/zip")
+            {
+                FileDownloadName = "WorkflowType.zip"
```

Comment:
Localize the file name.

Reply:
I adjusted the function signature so that if the exported workflow type is a single, the name of the workflow will be used automatically as the compressed package name, otherwise the localized string `WorkflowTypes.zip` will be used.

Reply:
But to be honest... This will look weird in Chinese, where in some cases there is no concept of plural, and if someone adds a translation to it , it will show up as a `工作流类型.zip`  
Of course, in this case we can just display the original content in the localized text

Reply:
Please change the code if you think it's necessary.